### ISSUE TITLE
CVE-2015-1538-1 : Error running the test, error running exec() ...

### ISSUE THREAD

**Issue by jeanbruder** [nowsecure/android-vts#39]
It would be nice to be able to read the whole content of the error message : can you add a popup window when pressing an error or the result of a test ?

**Comment by trufae** [comment]
Confirmed on Nexus9

**Comment by Fuzion24** [comment]
Related #27  #28   .. This app should not even be running on the N9.

**Comment by giantpune** [comment]
```sh
11-11 14:05:29.652  3332  3365 W System.err: java.io.IOException: Error running exec(). Command: [/data/user/0/com.nowsecure.android.vts/files/crashCheck-pie, 5, /data/user/0/com.nowsecure.android.vts/files/stagefrightCheck-pie, /data/user/0/com.nowsecure.android.vts/files/cve-2015-1538-1.mp4] Working Directory: null Environment: null
11-11 14:05:29.652  3332  3365 W System.err:    at java.lang.ProcessManager.exec(ProcessManager.java:211)
11-11 14:05:29.652  3332  3365 W System.err:    at java.lang.Runtime.exec(Runtime.java:174)
11-11 14:05:29.652  3332  3365 W System.err:    at java.lang.Runtime.exec(Runtime.java:129)
11-11 14:05:29.652  3332  3365 W System.err:    at fuzion24.device.vulnerability.vulnerabilities.helper.CrashCheck.execute(CrashCheck.java:23)
11-11 14:05:29.653  3332  3365 W System.err:    at fuzion24.device.vulnerability.vulnerabilities.framework.media.StageFright$1.isVulnerable(StageFright.java:76)
11-11 14:05:29.653  3332  3365 W System.err:    at fuzion24.device.vulnerability.test.VulnerabilityTestRunner.doInBackground(VulnerabilityTestRunner.java:56)
11-11 14:05:29.653  3332  3365 W System.err:    at fuzion24.device.vulnerability.test.VulnerabilityTestRunner.doInBackground(VulnerabilityTestRunner.java:14)
11-11 14:05:29.653  3332  3365 W System.err:    at android.os.AsyncTask$2.call(AsyncTask.java:295)
11-11 14:05:29.653  3332  3365 W System.err:    at java.util.concurrent.FutureTask.run(FutureTask.java:237)
11-11 14:05:29.653  3332  3365 W System.err:    at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:234)
11-11 14:05:29.653  3332  3365 W System.err:    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
11-11 14:05:29.653  3332  3365 W System.err:    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
11-11 14:05:29.653  3332  3365 W System.err:    at java.lang.Thread.run(Thread.java:818)
11-11 14:05:29.653  3332  3365 W System.err: Caused by: java.io.IOException: Permission denied
11-11 14:05:29.653  3332  3365 W System.err:    at java.lang.ProcessManager.exec(Native Method)
11-11 14:05:29.653  3332  3365 W System.err:    at java.lang.ProcessManager.exec(ProcessManager.java:209)
11-11 14:05:29.653  3332  3365 W System.err:    ... 12 more
11-11 14:05:29.653  3332  3365 D VULN_TEST: StageFright: cve-2015-1538-1  failed with: Error running exec(). Command: [/data/user/0/com.nowsecure.android.vts/files/crashCheck-pie, 5, /data/user/0/com.nowsecure.android.vts/files/stagefrightCheck-pie, /data/user/0/com.nowsecure.android.vts/files/cve-2015-1538-1.mp4] Working Directory: null Environment: null
1
```

**Comment by Fuzion24** [comment]
Fixed by 55e153b4f63a6a4091b5b210590de559a5d4e57f#diff-0956f308742b3fd693762651e2b0700bR38
